tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -1042,20 +1042,20 @@ def platform
 end
 
 # Check if this image is an I2 image
-@i2_image = nil # Cache the lookup result
-def nexus_i2_image
-  return @i2_image unless @i2_image.nil?
+@image = nil # Cache the lookup result
+def nexus_image
+  return @image unless @image.nil?
   on(agent, facter_cmd('-p cisco.images.system_image'))
-  @i2_image = stdout[/7.0.3.I2/] ? true : false
-  @i2_image
+  @image = stdout[/[A-Z]+\d/]
+  @image
 end
```

Comment:
👍 Nice one.

I'm also going to modify the code so that it can handle different FCS images (ex: D1.1 vs D1.2)

```diff
- @image ||= on(agent, facter_cmd('-p cisco.images.system_image'))
+ @image ||= on(agent, facter_cmd('-p cisco.images.system_image'))[/[A-Z]+\d\.\d+/]
```